relay/kusama/src/lib.rs

```diff
@@ -156,7 +156,7 @@ pub const VERSION: RuntimeVersion = RuntimeVersion {
 	spec_name: create_runtime_str!("kusama"),
 	impl_name: create_runtime_str!("parity-kusama"),
 	authoring_version: 2,
-	spec_version: 1_001_000,
+	spec_version: 1_001_003,
```

Comment:
```suggestion
	spec_version: 1_002_000,
```

@s0me0ne-unkn0wn why `1_001_003`? Are we going to hotfix this for `1.1.2`?
If this PR is going to the `main` and is expected to be release as a next `1.2.0` release, then I would suggest to change it to the `1_002_000` and also you could maybe change it for other runtimes as it is part of TODOs here: https://github.com/polkadot-fellows/runtimes/issues/140:
```
- [ ]  check and bump runtime/transaction versions
 ```

Reply:
I'm not familiar with how the versioning goes; I just needed it to be higher than `1.1.2` to pass [the idempotency check](https://github.com/polkadot-fellows/runtimes/pull/178/files#diff-efa4caeb17487ecb13d8f5eb7863c3241d84afa2e73fbf25909a2ca89df0f362R1782). Otherwise, the migration applies twice. I'll change it to `1.2.0` for both runtimes in this PR then. Thanks for pointing out!